src/background/global-action-creators/feature-flags-action-creator.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { Messages } from '../../common/messages';
+import { PREVIEW_FEATURES_TOGGLE } from '../../common/telemetry-events';
+import { FeatureFlagActions, FeatureFlagPayload } from '../actions/feature-flag-actions';
+import { Interpreter } from '../interpreter';
+import { TelemetryEventHandler } from '../telemetry/telemetry-event-handler';
+
+export class FeatureFlagsActionCreator {
+    constructor(
+        private readonly interpreter: Interpreter,
+        private readonly featureFlagActions: FeatureFlagActions,
+        private readonly telemetryEventHandler: TelemetryEventHandler,
+    ) {}
+
+    public registerCallback(): void {
```

Comment:
Kind of a nit, but should we have this be `registerCallbacks` (with an 's' at the end) to be consistent with the `GlobalActionCreator`?

Reply:
Good catch! We do want them to be consistent.